CMakeLists.txt

```diff
@@ -1587,8 +1587,20 @@ if(HEX_FILES_TO_MERGE)
   message(VERBOSE "Merging hex files: ${HEX_FILES_TO_MERGE}")
 endif()
 
-if(EMU_PLATFORM)
-  include(${ZEPHYR_BASE}/cmake/emu/${EMU_PLATFORM}.cmake)
+if(EMU_PLATFORM OR SUPPORTED_EMU_PLATFORMS)
```

Comment:
do we need both EMU_PLATFORM and SUPPORTED_EMU_PLATFORMS? Why not just move to a list type SUPPORTED_EMU_PLATFORMS in general and remove EMU_PLATFORM. Platforms with one supported type will only have 1 value in this case.

Reply:
Agree, just as proposed here as well: https://github.com/zephyrproject-rtos/zephyr/pull/38102#issuecomment-909150046

Reply:
Hi, this PR is currently using `EMU_PLATFORM` as a variable that enables choosing between different emulation platforms that are specified for a given board in the `SUPPORTED_EMU_PLATFORMS` list. Removing it would mean that we would need to introduce in this PR other mechanism that would allow such feature. Currently e.g. twister script through yaml file chooses which software to use in the CI by utilizing `EMU_PLATFORM` variable.